CAIPs/caip-169.md

```diff
@@ -0,0 +1,516 @@
+
+## Motivation
+<!--The motivation is critical for CAIP. It should clearly explain why the state of the art is inadequate to address the problem that the CAIP solves. CAIP submissions without sufficient motivation may be rejected outright.-->
+
+Web3 applications like DAOs, Defi, NFT market places etc. need verifiable
+offchain and onchain reputation to enable certain features for their end users.
+Web3 is missing a coherent method for requesting identity assertions from their
+users, e.g. for sign-in and sign-up. The majority of Web3 projects are using an
+approach where they cryptographically bind a signature produced by a
+single-context (and single-chain) wallet to the identity assertion; in the EVM,
+for example, these approaches usually rely on either
+[eth.personal.sign](https://web3js.readthedocs.io/en/v1.4.0/web3-eth-personal.html#sign)
+or [EIP-712](https://eips.ethereum.org/EIPS/eip-712). The identity assertion
+becomes self-certifiable with this approach, but tightly bound to the Ethereum
+address as the only identifier. To improve privacy it is important to introduce
+a mechanism that allows people to selectively disclose the linkage between
+another identifier (such as a chain-agnostic or off-chain DID) and their
+blockchain account address. 
+
+This can be done through VCs and DIDs. This CAIP introduces new JSON-RPC methods
+that are needed to build decentralized reputation for offchain and onchain uses
+alike.
+
+The identifiers by which issuers and holders and their key material are
+dereferenced for verification and binding purposes are out of scope of this
+specification, but complemented nicely by DID-based approaches such as those
+established in the EVM space by [EIP-2844][] or the cross-chain blockchain-based
+approach indexed by the [did pkh][] multicodec. Implementing verifiable
+credential exchange without DIDs may lead to a closed and limited system with
+limited interoperability or exportability; systems that can extend and federate
+over time are encouraged to build in flexibility at both the credential format
+level as well as the decentralized identifier level.
+
+## Specification
+
+Three new JSON-RPC methods are specified under the new `wallet_creds_*` prefix.
+
+### Verifiable Credential Proofs
+
+This section provides guidance on recommended [LD-Proof
+Suites](https://w3c-ccg.github.io/ld-proofs/) and [IANA JWS
+algorithm](https://www.iana.org/assignments/jose/jose.xhtml) support of embedded
+and external proofs for VCs.  The abbreviations for common VC formats in
+production today are taken from the [DIF Claim Format registry][] of the
+[Presentation Exchange][] meta-protocol specification governed by the
+Decentralized Identity Foundation, while the Verifiable Credentials profile
+defining their proof formats are specified in the [VC spec][] itself.
+
+#### Embedded Proofs
+
+CPs MUST support the following LD-Proof types for embedded proofs (i.e. VC-LDP):
```

Comment:
I didn't see CP defined anywhere, but perhaps it's well known?

Reply:
Whoops, missed that somehow.  Oliver's original draft referred to VC issuance as a "Credential Provider" when this was drafted as a potential EIP; it's fairly standard terminology in the RPC-based dapp<>wallet<>node transport model of EVM